common/src/main/java/com/wynntils/models/character/CharacterPhysicalModel.java

```diff
@@ -0,0 +1,50 @@
+package com.wynntils.models.character;
+
+import com.wynntils.core.WynntilsMod;
+import com.wynntils.core.components.Handlers;
+import com.wynntils.core.components.Model;
+import com.wynntils.handlers.bossbar.event.BossBarAddedEvent;
+import com.wynntils.mc.event.TickEvent;
+import com.wynntils.models.character.bossbar.DeathScreenBar;
+import com.wynntils.models.character.event.CharacterDeathEvent;
+import com.wynntils.models.character.event.CharacterMovedEvent;
+import com.wynntils.utils.mc.McUtils;
+import com.wynntils.utils.mc.type.Location;
+import net.minecraft.core.Position;
+import net.neoforged.bus.api.SubscribeEvent;
+
+import java.util.List;
+
+public final class CharacterPhysicalModel extends Model {
+    private static final DeathScreenBar deathScreenBar = new DeathScreenBar();
```

Comment:
I see where you are going with this. You want `CharacterModel` to the model of `semi-static` character info, like class type, level, id, etc. and want this model to be more of a `CharacterStateModel` handling whatever the player can do with the character dynamically during the gameplay. I think this is acceptable as CharacterModel outgrew itself (although extracting Rank knowledge already made it pretty much okay).